### PR TITLE
feat: added more info to content single and analytics for buttons

### PR DESCRIPTION
**Pull request by nlewis84**
This PR adds potential parent node info to content single analytics and also tracks button presses. I tried to follow similar patterns already in the code base. 

~~@vinnyjth or @redreceipt I could use a hand with testing these out in Amplitude, and I'm going to mark this as a draft PR in case we want to adjust how I have gone about things with this one.~~

## Screenshot
Proof that the events are coming through to Amplitude
Screen Shot 2021-12-02 at 10 55 39 AM

## Testing
1. Open Flipper
1. Open a content item with a button in your simulator
1. Click the button in the simulator
1. Check your network requests in Flipper (search for trackEvent)
Screen Shot 2021-12-02 at 11 05 42 AM
1. Select the most recent request in the filtered list
1. You should see a response like this if the event was tracked
Screen Shot 2021-12-02 at 11 05 29 AM

### CODE REVIEW

File: packages/apollos-ui-connected/src/features/ButtonFeature.js [hunk 1]
```diff
@@ -2,10 +2,16 @@ import React from 'react';
...
-import { Button, PaddedView } from '@apollosproject/ui-kit';
+import { withTrackOnPress } from '@apollosproject/ui-analytics';
+import { Button, PaddedView, styled } from '@apollosproject/ui-kit';
 
 const ButtonFeature = ({ action }) => {
-  const { data } = useQuery(
+  const ButtonWithAnalytics = styled(
+    {},
+    'ui-connected.features.ButtonFeature'
+  )(withTrackOnPress(Button));
```
Comment (conrad-vanl) [inline packages/apollos-ui-connected/src/features/ButtonFeature.js:12]: I hav a feeling you'll need to pass these props https://github.com/ApollosProject/apollos-apps/blob/3ea9d4b1de4d5dd01f3b3caec4256e57e8ab8f84/packages/apollos-ui-analytics/src/withTrackOnPress.js#L7-L8 to this component when it's rendered now.

@redreceipt @vinnyjth any of y'all remember how this works?

**Comment by nlewis84** [comment]
This is primarily for VCC's v2 deadline on December 10.

**Review by redreceipt — APPROVED** [review]

**Comment by nlewis84** [comment]
@redreceipt These are the tests (data-connector-postgres) that are currently broken, right? Am I okay to admin merge here?

**Comment by redreceipt** [comment]
I'm gonna try an alternate method first to fix the tests than the existing PR.

**Pull request merged**